File: app/src/main/java/com/bombadu/tabbly5/TabThree.java
package com.bombadu.tabbly5;

import android.graphics.Color;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

public class TabThree extends Fragment {
    boolean isTextColor = false;
    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.tab3, container, false);

        final TextView threeTextView = view.findViewById(R.id.threeTextView);
        threeTextView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (!isTextColor){
                    threeTextView.setTextColor(Color.BLUE);
                    isTextColor = true;
                } else {
                    threeTextView.setTextColor(Color.BLACK);
                    isTextColor = false;
                }
            }
        });
        return view;
    }
}

File: app/src/main/java/com/bombadu/tabbly5/TabThree.kt
package com.bombadu.tabbly5

import android.graphics.Color
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment

class TabThree : Fragment() {
    var isTextColor = false
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.tab3, container, false)
        val threeTextView = view.findViewById<TextView>(R.id.threeTextView)
        threeTextView.setOnClickListener {
            isTextColor = if (!isTextColor) {
                threeTextView.setTextColor(Color.BLUE)
                true
            } else {
                threeTextView.setTextColor(Color.BLACK)
                false
            }
        }
        return view
    }
}